packages/terra-card/src/Card.scss

```diff
@@ -0,0 +1,27 @@
+
+/* stylelint-disable selector-class-pattern  */
+.card {
```

Comment:
I'd say we only make theme-able what needs to be. Its fairly trivial to expose additional properties.